experiments/bistatic_test.py

```diff
@@ -28,22 +28,28 @@ class BistaticTest(ExperimentPrototype):
     def __init__(self, **kwargs):
         """
         kwargs:
-
-        freq: int, kHz
-
+            listen_to: str, one of the three-letter site codes. e.g. listen_to='cly'
+            beam_order: str, beam order for tx. Only used if listen_to not specified. Format as '1,3,5,6-10',
+                which will use beams [1, 3, 5, 6, 7, 8, 9, 10]
         """
         cpid = 3820
-        super().__init__(cpid)
 
         num_ranges = scf.STD_NUM_RANGES
-        if scf.opts.site_id in ["cly", "rkn", "inv"]:
-            num_ranges = scf.POLARDARN_NUM_RANGES
+
+        common_freqs = {            # copied from superdarn_common_fields.py - September 2022
+            'sas': [10500, 13000],
+            'pgr': [10600, 13100],
+            'rkn': [10900, 12300],
+            'inv': [10800, 12200],
+            'cly': [10700, 12500]
+        }
 
         # default frequency set here
-        if scf.opts.site_id in ['cly', 'inv', 'sas']:
-            freq = scf.COMMON_MODE_FREQ_1
-        else:   # RKN and PGR listen to CLY
-            freq = 10700    # CLY freq 1
+        listen_to = kwargs.get('listen_to', scf.opts.site_id)   # If 'listen_to' specified, tune in to that radar
+        if listen_to not in common_freqs.keys():
+            raise ValueError('Not a valid site ID: {}'.format(listen_to))
+
+        freq = common_freqs.get(listen_to)[0]
```

Comment:
I'm not sure. I figured it would be better to listen than to transmit if the user specified something wrong, lest transmitting mess with the other radars. For me, I would want an error to be raised when I schedule the radar, rather than find out however many days later that my experiment isn't running how it's supposed to. So perhaps we should consider adding in checks to the scheduler for this.

Reply:
I'll add it to the trello card.